src/ray/gcs/accessor.h

```diff
@@ -551,6 +551,9 @@ class NodeInfoAccessor {
       const std::shared_ptr<rpc::HeartbeatTableData> &data_ptr,
       const StatusCallback &callback) = 0;
 
+  /// Resend heartbeat when GCS restarts from a failure.
+  virtual void AsyncReReportHeartbeat() = 0;
```

Comment:
virtual void AsyncReReportHeartbeat() {} ?

Reply:
Since classes in accessor.h are all interfaces, i think it'll be better to keep it pure virtual, consistent with other functions to avoid potential confusing.
In future if we remove redis accessor, all need to do is removing the file.